### PR TITLE
Change network correctly in top trending tokens widget

### PR DESCRIPTION
**Pull request by nguyenhuudungz**

### CODE REVIEW

File: src/pages/TrueSight/hooks/useGetTrendingSoonTokenId.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export default function useGetTrendingSoonTokenId(token?: Token): number | undef
     }
 
     asyncF()
-  }, [token])
+  }, [JSON.stringify(token)])
```
Comment (viet-nv) [inline src/pages/TrueSight/hooks/useGetTrendingSoonTokenId.ts:32]: token0, token1 were memorize by useMemo, so why do you need stringify here?

Comment (nguyenhuudungz) [reply src/pages/TrueSight/hooks/useGetTrendingSoonTokenId.ts:32]: I know, but somehow it just re-render... It takes time to find the root causes.

Comment (nguyenhuudungz) [reply src/pages/TrueSight/hooks/useGetTrendingSoonTokenId.ts:32]: I solve it.

File: src/pages/TrueSight/hooks/useGetTrendingSoonTokenId.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ export default function useGetTrendingSoonTokenId(token?: Token): number | undef
   useEffect(() => {
     const asyncF = async () => {
       setTokenId(undefined)
+      console.log(`I'm here:`, token?.symbol)
```
Comment (viet-nv) [inline src/pages/TrueSight/hooks/useGetTrendingSoonTokenId.ts:13]: remove console.log bro

**Review by viet-nv — APPROVED** [review]

**Pull request merged**